bot/cogs/filtering.py

```diff
@@ -184,12 +184,25 @@ def mod_log(self) -> ModLog:
                         else:
                             channel_str = f"in {msg.channel.mention}"
 
+                        # Word and match stats for watch_words and watch_tokens
+                        if filter_name in ("watch_words", "watch_tokens"):
+                            m = triggered
+                            match = m[0]
```

Comment:
This is a bit weird cause it could just be `match = triggered[0]`. Maybe you did it that way because the name `triggered` is misleading. In that case, it could just be renamed to something like `match` anyway.

Reply:
I do suppose I can rename `triggered` to `match`, since they're functionally the same thing and would provide clarity in certain areas. What do you think?

Reply:
Yes, that sounds good.